bundle-size/test/api.test.js

```diff
@@ -33,25 +33,29 @@ describe('bundle-size api', () => {
   let probot;
   let app;
   const db = dbConnect();
-  const nodeCache = new NodeCache();
+  let nodeCache;
+  let logWarnSpy;
 
-  beforeAll(async () => {
+  beforeAll(async done => {
     await setupDb(db);
 
     probot = new Probot({});
     app = probot.load(app => {
       const githubUtils = new GitHubUtils(new Octokit(), app.log, nodeCache);
       installApiRouter(app, db, githubUtils);
     });
+    // Stub app.log.warn to silence test log noise
+    logWarnSpy = jest.spyOn(app.log, 'warn').mockImplementation();
 
     // Return a test token.
     app.app = {
       getInstallationAccessToken: () => Promise.resolve('test'),
     };
+    done();
   });
 
-  beforeEach(async () => {
-    nodeCache.flushAll();
+  beforeEach(() => {
+    nodeCache = new NodeCache();
```

Comment:
🤔

Reply:
`async` was not necessary; there's no `await`.
The `nodeCache` instance was leaving dangling `setTimeouts` the way it was being reused (Jest gave me warnings about this), and there's no need to share a cache between the tests